libs/schema-registry/src/lib/create-pr-request/value.ts

```diff
@@ -1,9 +1,24 @@
-import { EnumPullRequestMode, GitResourceMeta } from "@amplication/git-utils";
+import {
+  EnumGitProvider,
+  EnumPullRequestMode,
+  GitResourceMeta,
+} from "@amplication/git-utils";
 import { IsOptional, IsString, ValidateNested } from "class-validator";
-import { EnumGitProvider } from "../../models";
-import { CreateCommit } from "./create-commit.dto";
 
-export class CreatePullRequestArgs {
+class Commit {
+  @IsString()
+  @IsOptional()
+  base?: string | undefined;
+  @IsOptional()
+  @IsString()
+  head?: string | undefined;
```

Comment:
@tupe12334 / @abrl91 Why do we need `base` and `head`? `amplication-server` never produce an event passing these values so I guess it's safe to remove it.

Reply:
@yuval-hazaz Wanted to leave that in case we want to do a future implementation; I agree with you but it's up to @yuval-hazaz